### PR TITLE
fix bastion scripts

### PR DESCRIPTION
**Pull request by pranavgaikwad**

### CODE REVIEW

File: files/prepare_station.sh [hunk 1]
```diff
@@ -105,7 +105,10 @@ get_cluster_info(){
     if sshpass -p "$PASSWORD" scp $STUDENT@$HOSTNAME:./cluster.info cluster.ocp3
     then
         printf "Grabbing cluster info from OCP3 cluster\n"
-        cp cluster.info cluster.orig
+        if [ -f cluster.orig ]; then
+	  cp cluster.orig cluster.info
+	fi 
+    	cp cluster.info cluster.orig
```
Comment (kaovilai) [inline files/prepare_station.sh:111]: Why are there two cp?

Comment (pranavgaikwad) [reply files/prepare_station.sh:111]: makes it idempotent on multiple logins

Comment (kaovilai) [reply files/prepare_station.sh:111]: If I read this right..

if cluster.org exists, then copy it to cluster.info
then copy cluster.info to cluster.orig

This means cluster.orig takes priority if it exists.. and cluster.info data prior to line 108 is discarded.

Comment (pranavgaikwad) [reply files/prepare_station.sh:111]: cluster.orig does not exist the very first time, it only has one section for OCP4

**Review by kaovilai — APPROVED** [review]

**Pull request merged**